src/v2/ion/i18nTransformer.js

```diff
@@ -122,6 +122,35 @@ const I18N_OVERRIDE_MAPPINGS = {
   'oie.session.expired' : 'oie.idx.session.expired',
 };
 
+const getDisplayName = (remediation, defaultName) => {
+  return remediation.relatesTo?.value?.displayName || defaultName;
+};
+
+const I18N_PARAMS_MAPPING = {
+  [FORMS.ENROLL_AUTHENTICATOR]: {
+    [AUTHENTICATOR_KEY.ON_PREM]: {
+      getParam: getDisplayName,
+      defaultParam: loc('oie.on_prem.authenticator.default.vendorName', 'login'),
+    },
+    [AUTHENTICATOR_KEY.RSA]: {
+      getParam: getDisplayName,
+      defaultParam: loc('oie.on_prem.authenticator.default.vendorName', 'login'),
+    },
+  },
```

Comment:
As we add more enrollment forms, this dictionary is going to be a burden to manage. Any reason we can't apply this to all forms now?

Reply:
+1 Should Custom OTP also be added to this list ? https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L829

Reply:
If I understand this correct, we have the same logic to show a vendor name by reading `remediation.relatesTo?.value?.displayName` and  fallback to show a  default vendor name. 
 
This is  also only for a few use cases where we have vendor specific authenticators like (on prem, rsa, custom OTP) but the logic is the same for all.

Reply:
> As we add more enrollment forms, this dictionary is going to be a burden to manage. Any reason we can't apply this to all forms now?

I tried that first (to just pass displayName as a param always), but i cannot set params on a string that does not need them. courage throws an error - https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/StringUtil.js#L64

Reply:
> +1 Should Custom OTP also be added to this list ? https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L829

The only reason i cannot is because im not sure if the uischema.label for custom totp string takes a param

Reply:
Hmm not sure if I understand that. But can we not tell by looking at this ? https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L829

Reply:
Right but none of the strings (except title https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L826) takes a param